docs/baking.md

```diff
@@ -192,6 +192,10 @@ sudo -u tezos tezos-client register key baker as delegate
 Check a blockchain explorer (e.g. https://tzkt.io/ or https://tzstats.com/) to see the baker status and
 baking rights of your account.
 
+### Using ledger on WSL
+
+In order to use ledger on WSL some additional preparation steps are needed. We recommed to use `usbipd-win` tool and follow [this guide](https://docs.microsoft.com/en-us/windows/wsl/connect-usb) about connecting usb devices to WSL.
+
```

Comment:
This seems like an odd place for this info to me because:
1. it comes after the commands that would have used the ledger itself
2. this guide is the one based on `systemd` which as we now know isn't supported officially, so many people won't look for this info here

I believe it would be better place inside the (missing) section that I mentioned two comments above instead.

Reply:
Yeah, I agree that this place indeed looks odd.

> I believe it would be better place inside the (missing) section that I mentioned two comments above instead.

Putting this comment in that section would indeed be better, but probably still not the most appropriate, because it contains information about installing packages, rather than using them :thinking:

Reply:
well, most of the information in that Ubuntu document is in fact about installation, so I think this fits there too.

Admittedly this explains both what to install and where to find docs on how to use that, but I think that's fine (and there is no better place for that).